src/App/Components/Common/NavProfile/Profile.js

```diff
@@ -0,0 +1,127 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import isEmpty from 'lodash/isEmpty';
+import PropTypes from 'prop-types';
+import ReactImageFallback from 'react-image-fallback';
+import actions from '../../../../Redux/Actions/notifications';
+import defaultData from '../../../../configs/urls';
+import TriangularPopup from '../TriangularPopup';
+
+const { show, hide } = actions;
+
+export class Profile extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isShown: false,
+      profile: {},
+      notifications: [],
+      notificationsBubble: ''
+    };
+  }
+
+  componentWillReceiveProps({
+    isShown, profile, configs, data
+  }) {
+    this.setState({ isShown });
+    if (!isEmpty({ ...profile, ...configs })) this.setState({ profile });
+    else this.setState({ profile: {} });
+    if (!isEmpty(data)) {
+      this.setState({ notifications: data });
+      if (data.length < 10) {
+        this.setState({ notificationsBubble: `${data.length}` });
+      } else this.setState({ notificationsBubble: '9+' });
```

Comment:
@nkpremices lol, you know this is a hack right? Why not display the exact number of notifications?